### ISSUE TITLE
create dashboard view

### ISSUE THREAD

**Issue by lcabrini** [borderitsolutions/amadaa#74]

**Comment by BorderITS** [comment]
ok i will review and comment soon

**Comment by BorderITS** [comment]
It's working ! fine good job but if you can make the customization of the login page at the user login that will be really cool

**Comment by lcabrini** [comment]
Yes, I will. But this issue is related to the dashboard, so let's take one thing at a time.
